client/src/components/ManageBookings/BookModule/CurrentBookings.tsx

```diff
@@ -14,9 +14,9 @@ const CurrentBookings = ({ bookings, handleAccept }: Props): JSX.Element => {
   const now = new Date();
 
   let pastBookings = [...bookings];
-  pastBookings = pastBookings.filter((request) => request.date < now);
+  pastBookings = pastBookings.filter((request) => request.day < now);
   let futureBookings = [...bookings];
-  futureBookings = futureBookings.filter((request) => request.date >= now);
+  futureBookings = futureBookings.filter((request) => request.day >= now);
 
```

Comment:
Ahh yeah that's fair, they're saved as requests (when the request for a booking is made) rather than bookings so I thought this made more sense, but I can see how this is confusing.